### DISCUSSION TITLE
Darktehme unreadable text on firefox

### DISCUSSION THREAD

**Discussion by SylvainMartel** [community/community#32347] [Other Feature Feedback, Questions, & Ideas] [answered]
Since this morning, the github dark theme doesn't render correctly on firefox, making the text illisible.  See the attached file, the text is dark on dark.  Nothing was changed in frefox, still using 104.0.2

![image](https://user-images.githubusercontent.com/9009158/189221748-284f7af6-c892-4a3e-91bd-3c8d7dc467d8.png)

**Comment by softorangetech1122** [comment]
You can make an issue on the Primer CSS (GitHub's design framework). I may be off-base, however there ought to be an alternate group for Primer CSS there.

**Comment by vasvir** [comment, marked as answer]
Hey firefox dark theme here.

If I disable the facebook container it works. I found it here: https://news.ycombinator.com/item?id=32771555

**Comment by SylvainMartel** [comment]
Wow, that was indeed the facebook container.  Weird.  2.3.4 fixes it, but I had to remove the previous version manually.  Not a good release.
